rag_experiment_accelerator/config/tests/test_index_config.py

```diff
@@ -33,29 +33,33 @@ def test_index_config_to_index_name():
 
 
 def test_index_name_to_index_config():
```

Comment:
Severity: Medium - Consider using constants or enums for configuration values to avoid magic values like True or False scattered throughout your tests. This improves readability and maintainability.


[Hack - peer-review-pal] We are sorry for the intrusion. Feel free to ignore/delete the comments with the tag - they are automatically generated by [our Hack project](https://github.com/microsoft/peer-review-pal).
If you find them useful, want us to delete them or have any feedback or know how to make it better and want to contribute - reach out to some of us: juliame@microsoft.com vadimkirilin@microsoft.com oteslenko@microsoft.com vimeena@microsoft.com
You can prevent future comments from appearing by adding #prp-ignore to your PR description.